ddtrace/contrib/cassandra/session.py

```diff
@@ -166,14 +105,46 @@ def _extract_result_metas(result):
 
     return metas
 
-def _sanitize_query(query):
+def _sanitize_query(span, query):
     """ Sanitize the query to something ready for the agent receiver
     - Cast to unicode
     - truncate if needed
     """
     # TODO (aaditya): fix this hacky type check. we need it to avoid circular imports
-    if type(query).__name__ in ('SimpleStatement', 'PreparedStatement'):
+    t = type(query).__name__
+
+    resource = None
+    if t in ('SimpleStatement', 'PreparedStatement'):
         # reset query if a string is available
-        query = getattr(query, "query_string", query)
+        resource = getattr(query, "query_string", query)
+    elif t == 'BatchStatement':
+        resource = 'BatchStatement'
+        q = "; ".join(q[1] for q in query._statements_and_parameters[:2])
+        span.set_tag("cassandra.query", q)
+        span.set_metric("cassandra.batch_size", len(query._statements_and_parameters))
+    elif t == 'BoundStatement':
+        ps = getattr(query, 'prepared_statement', None)
+        if ps:
+            resource = getattr(ps, 'query_string', None)
+    else:
+        resource = 'unknown-query-type' # FIXME[matt] what else do to here?
```

Comment:
looks like this triggers on the not newest version of the lib. will add more testing.